### PR TITLE
Add tests for Disable syntax for scala 3

### PR DESCRIPTION
**Pull request by mlachkar**

### CODE REVIEW

File: scalafix-tests/input/src/main/scala-3/test/disableSyntax/DisableSyntaxFinalVal.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*
+rules = DisableSyntax
+DisableSyntax.noFinalVal = true
+ */
+package test
```
Comment (bjaglin) [inline scalafix-tests/input/src/main/scala-3/test/disableSyntax/DisableSyntaxFinalVal.scala:5]: inconsistent with directory

File: scalafix-tests/input/src/main/scala-3/test/disableSyntax/DisableSyntaxInstanceOf.scala [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/*
+rules = DisableSyntax
+DisableSyntax.noIsInstanceOf = true
+DisableSyntax.noAsInstanceOf = true
+ */
+package test
```
Comment (bjaglin) [inline scalafix-tests/input/src/main/scala-3/test/disableSyntax/DisableSyntaxInstanceOf.scala:6]: inconsistent with directory

File: scalafix-tests/input/src/main/scala-3/test/disableSyntax/DisableSyntaxMoreRules.scala [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+DisableSyntax.noContravariantTypes = true
+DisableSyntax.noDefaultArgs = true
+DisableSyntax.noValInAbstract = true
+DisableSyntax.noImplicitObject = true
+DisableSyntax.noImplicitConversion = true
+DisableSyntax.noUniversalEquality = true
+DisableSyntax.noUniversalEqualityMessage =
+  "== and != are not typesafe, use === and =!= from cats.Eq instead"
+*/
+package test
```
Comment (bjaglin) [inline scalafix-tests/input/src/main/scala-3/test/disableSyntax/DisableSyntaxMoreRules.scala:13]: inconsistent with directory

File: build.sbt [hunk 4]
```diff
@@ -155,9 +153,7 @@ lazy val testsOutput = project
                          Seq("-P:semanticdb:synthetics:on")
                        else Nil),
     scalacOptions -= warnUnusedImports.value,
-    libraryDependencies ++= (if (scalaVersion.value.startsWith("2"))
-                               testsDeps :+ "org.scala-lang" % "scala-reflect" % scalaVersion.value
-                             else Nil),
+    libraryDependencies ++= testDependencies.value,
```
Comment (bjaglin) [inline build.sbt:156]: this is about the tests module, not the test configuration, so I think plural matters
```suggestion
    libraryDependencies ++= testsDependencies.value,
```

**Review by bjaglin — APPROVED** [review]
Great to see some coverage. Do we really need to keep so many files separated for scala-2 and scala-3 though? a quick look shows that there are very similar

**Comment by mlachkar** [comment]
We are not oblige to keep everything seperates. I thought about this too. let's group them again. there is only one difference.

**Pull request merged**